app/javascript/flavours/glitch/components/status.js

```diff
@@ -251,6 +251,10 @@ class Status extends ImmutablePureComponent {
     // as it could cause surprising changes when receiving notifications
     if (settings.getIn(['content_warnings', 'shared_state']) && status.get('spoiler_text').length && !status.get('hidden')) return;
 
+    let autoCollapseHeight = parseInt(autoCollapseSettings.get('height'));
+    if (status.get('media_attachments').size && !muted) {
+      autoCollapseHeight += 210;
```

Comment:
```suggestion
      autoCollapseHeight += 250;
```
This gets set, but never used anywhere.
Also shouldn't this be '250' to match the current default?

Reply:
Wanted to follow up on this one:
210 is the actual computed default height after all is said and done for previews. When it was at 250 there were posts with a status__content_text area that was > 500px (the setting for my instance, as seen above) 

Happy to restore it to 250px when I fix this up but IMO behavior is closer to expectations at 210 :)